pkg/env/env.go

```diff
@@ -17,3 +24,27 @@ const (
 	// CustomerID is the env var we look in for a Customer ID.
 	CustomerID = "FASTLY_CUSTOMER_ID"
 )
+
+func Vars() []string {
+	vars := []string{}
+	for _, e := range os.Environ() {
+		pair := strings.SplitN(e, "=", 2)
+		vars = append(vars, toVar(pair[0]))
+	}
+	return vars
+}
+
+func toVar(v string) string {
+	if runtime.GOOS == "windows" {
```

Comment:
For consistency could we replace this with https://github.com/fastly/cli/blob/main/pkg/runtime/runtime.go#L11